### DISCUSSION TITLE
Using Retro-Bit's RetroPort Adapter to dump NES Games

### DISCUSSION THREAD

**Discussion by PW5190** [sanni/cartreader#584] [General Chat]
Hey, I have a question. I know Save the Hero Builders made an NES adapter for the Open-Source Cartridge Reader that plugs into the SNES slot, but has anyone tested out the RetroPort Adapter (a Retro-Bit device that allows NES games to be played on the SNES, but only uses its own AV source)? Will it work the same way as the StHB NES adapter? I don't have the RetroPort or the OSCR to test for myself, so I can't validate.

**Comment by PW5190** [comment]
And I know very well the OSCR had Mega Drive and Game Boy slots, so using the Super Game Boy, Super Game Boy 2, RetroGen and Super Retro Advance adapters would obviously be out of the question for me. (Unless you can dump the firmware ROMs from the former two using the OSCR, but I wouldn't count on it.)

**Comment by sanni** [comment]
The RetroPort Adapter will not work as it's basically a whole NES-on-a-Chip console not just a passive adapter.
